### ISSUE TITLE
Fixture data does not load

### ISSUE THREAD

**Issue by worthwhileindustries** [api-platform/demo#96]
Upping the environment and running `docker-compose exec php bin/console hautelook:fixtures:load` results in these errors:

```
$ docker-compose exec php bin/console hautelook:fixtures:load
Careful, database will be purged. Do you want to continue y/N ?y
2019-01-23T21:10:41+00:00 [error] Error thrown while running command "hautelook:fixtures:load". Message: "An exception occurred while executing 'SELECT UUID_GENERATE_V4()':

SQLSTATE[42883]: Undefined function: 7 ERROR:  function uuid_generate_v4() does not exist
LINE 1: SELECT UUID_GENERATE_V4()
               ^
HINT:  No function matches the given name and argument types. You might need to add explicit type casts."

In AbstractPostgreSQLDriver.php line 79:
                                                                               
  An exception occurred while executing 'SELECT UUID_GENERATE_V4()':           
                                                                               
  SQLSTATE[42883]: Undefined function: 7 ERROR:  function uuid_generate_v4()   
  does not exist                                                               
  LINE 1: SELECT UUID_GENERATE_V4()                                            
                 ^                                                             
  HINT:  No function matches the given name and argument types. You might nee  
  d to add explicit type casts.                                                
                                                                               

In PDOConnection.php line 90:
                                                                               
  SQLSTATE[42883]: Undefined function: 7 ERROR:  function uuid_generate_v4()   
  does not exist                                                               
  LINE 1: SELECT UUID_GENERATE_V4()                                            
                 ^                                                             
  HINT:  No function matches the given name and argument types. You might nee  
  d to add explicit type casts.                                                
                                                                               

In PDOConnection.php line 88:
                                                                               
  SQLSTATE[42883]: Undefined function: 7 ERROR:  function uuid_generate_v4()   
  does not exist                                                               
  LINE 1: SELECT UUID_GENERATE_V4()                                            
                 ^                                                             
  HINT:  No function matches the given name and argument types. You might nee  
  d to add explicit type casts.                                                
                                                                               

hautelook:fixtures:load [-b|--bundle [BUNDLE]] [-m|--manager MANAGER] [--append] [--shard SHARD] [--purge-with-truncate] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
```

**Comment by worthwhileindustries** [comment]
https://stackoverflow.com/questions/22446478/extension-exists-but-uuid-generate-v4-fails

This fixes it...

The extension is available but not installed in this database.

`CREATE EXTENSION IF NOT EXISTS "uuid-ossp";`

**Comment by dunglas** [comment]
You have to run Doctrine migrations first (we’ll automate that at some point).

**Comment by worthwhileindustries** [comment]
@dunglas ok yeah I actually just noticed it in the migrations as I was auditing the code. I think it would be just as good to add that to the readme. If I get a minute I'll make a PR. Thanks.

**Comment by vincentchalamon** [comment]
Hi @worthwhileindustries,

The db-generated uuid support has been [deprecated from doctrine/dbal](https://github.com/doctrine/dbal/blob/master/UPGRADE.md#deprecated-usage-of-db-generated-uuids) and will be removed very soon from this project in PR #95. Uuids will be generated using `ramsey/uuid-doctrine`.

Also, the db docker-compose image will be replaced by [`bitnami/postgresql`](https://hub.docker.com/r/bitnami/postgresql/) to be iso-prod. This new configuration will be implemented in PR #95, feel free to have a look at it :-)
